engine/src/mbliphonedc.mm

```diff
@@ -1203,6 +1244,11 @@ static void MCIPhoneDoDidBecomeActive(void *)
 	t_init_success = X_init(1, args, env);
 	
 	[t_pool release];
+    
+    // Release the env array and its elements
+    for (uindex_t j = 0; j < envc; ++j)
```

Comment:
Careful here - check what X_init actually does with env... It might expect it to still be around for the lifetime of the engine!